.github/workflows/build.yml

```diff
@@ -42,6 +42,9 @@ jobs:
 
     steps:
       - uses: actions/checkout@v4
+      - name: Run oxlint
+        # oxlint is fast, so wrong code will fail quickly
+        run: npx --yes oxlint@latest .
```

Comment:
Use `yarn dlx` because in this way the downloaded `oxc` package can be cached by `actions/setup-node`

Reply:
done